lib/bundlex/toolchain/xcode.ex

```diff
@@ -2,7 +2,9 @@ defmodule Bundlex.Toolchain.XCode do
   @moduledoc false
 
   use Bundlex.Toolchain
-  alias Bundlex.Toolchain.Common.Unix
+  alias Bundlex.Toolchain.Common.{Unix, Compilers}
+
+  @compilers %Compilers{c: "cc", cpp: "g++"}
```

Comment:
It seems on mac the best way is to use `cc` for both c and c++, as `man cc` says
> clang - the Clang C, C++, and Objective-C compiler